config/diracos.json

```diff
@@ -697,6 +697,45 @@
                   "name": "PyQt4"
                }
             ]
+         },
+         {
+            "name": "singularity",
+            "packages": [
+               {
+                  "comment": "Older go is required to build newer golang versions and prebuilt RPM is needed to avoid bootstrapping issues",
+                  "src": "http://springdale.math.ias.edu/data/puias/computational/6/x86_64//golang-1.5.1-0.sdl6.x86_64.rpm",
+                  "name": "golang-old",
+                  "buildOnly": true
+               },
+               {
+                  "comment": "Dependency of golang-old",
+                  "src": "http://springdale.math.ias.edu/data/puias/computational/6/x86_64//golang-bin-1.5.1-0.sdl6.x86_64.rpm",
+                  "name": "golang-bin-old",
+                  "buildOnly": true
+               },
+               {
+                  "comment": "Dependency of golang-old",
+                  "src": "http://springdale.math.ias.edu/data/puias/computational/6/x86_64//golang-src-1.5.1-0.sdl6.noarch.rpm",
+                  "name": "golang-src-old",
+                  "buildOnly": true
+               },
+               {
+                  "comment": "For running golang",
+                  "src": "http://vault.centos.org/6.10/os/Source/SPackages/mercurial-1.4-5.el6_9.src.rpm",
```

Comment:
Is this really necessary? Can't you just remove mercurial in the go spec file?

Reply:
For some reason I thought it was but it seems not: https://gitlab.cern.ch/CLICdp/iLCDirac/diracos-test/-/jobs/6473102

Fixed.